source/includes/_encryption.md

```diff
@@ -1,12 +1,44 @@
 # Encryption
 
-## Get Skykey
+
+
+## Getting a Skykey
 
 ```shell
 curl -A "Sia-Agent"  -u "":<apipassword> --data "name=key_to_the_castle" "localhost:9980/skynet/skykey"
 curl -A "Sia-Agent"  -u "":<apipassword> --data "id=gi5z8cf5NWbcvPBaBn0DFQ==" "localhost:9980/skynet/skykey"
 ```
 
+```go
```

Comment:
Not sure whether it's custom to be this verbose in `go` snippets?  
Do we want to be this verbose? I don't really mind, but I like snippets super short myself, or at least as short as possible.